### PR TITLE
Added basic buffer protocol

### PR DESCRIPTION
**Pull request by shavkunov**

### CODE REVIEW

File: src/main/proto/buffer.proto [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    limitations under the License.
+
+    Author: Mikhail Shavkunov (@shavkunov)
+ */
+syntax = "proto3";
+
+option java_package = "com.bardsoftware.backend.fleet.rmarkdown";
+option java_outer_classname = "BufferProtocol";
+
+message Buffer {
```
Comment (dbarashev) [inline src/main/proto/buffer.proto]: Please rename it to `CompilerFleetRequest`, the file to `fleet.proto` and outer class to `CompilerFleet`

File: build.gradle [hunk 2]
```diff
@@ -26,6 +28,25 @@ dependencies {
     compile "com.xenomachina:kotlin-argparser:2.+"
     compile "com.spotify:docker-client:8.11.+"
     compile "com.google.cloud:google-cloud-pubsub:0.32.0-beta"
+    compile 'com.google.protobuf:protobuf-java:3.5.+'
+    compile 'io.grpc:grpc-protobuf:1.+'
```
Comment (dbarashev) [inline build.gradle]: This is likely to become needed soon, but not now. Please comment it out and add explanation: "uncomment when we start doing grpc requests"

Comment (shavkunov) [reply build.gradle]: Done

File: src/main/proto/buffer.proto (hunk 1 above)
Comment (shavkunov) [reply src/main/proto/buffer.proto]: Done

**Review by dbarashev — APPROVED** [review]
LGTM

**Comment by dbarashev** [comment]
Update issue #5

**Pull request merged**